src/test/java/org/isf/medicalsinventory/test/Tests.java

```diff
@@ -0,0 +1,236 @@
+/*
+ * Open Hospital (www.open-hospital.org)
+ * Copyright © 2006-2024 Informatici Senza Frontiere (info@informaticisenzafrontiere.org)
+ *
+ * Open Hospital is a free and open source software for healthcare data management.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * https://www.gnu.org/licenses/gpl-3.0-standalone.html
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+package org.isf.medicalsinventory.test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+
+import org.isf.OHCoreTestCase;
+import org.isf.medicalinventory.model.MedicalInventory;
+import org.isf.medicalinventory.model.MedicalInventoryRow;
+import org.isf.medicalinventory.service.MedicalInventoryIoOperation;
+import org.isf.medicalinventory.service.MedicalInventoryIoOperationRepository;
+import org.isf.medicalinventory.service.MedicalInventoryRowIoOperation;
+import org.isf.medicalinventory.service.MedicalInventoryRowIoOperationRepository;
+import org.isf.medicals.model.Medical;
+import org.isf.medicals.service.MedicalsIoOperationRepository;
+import org.isf.medicals.test.TestMedical;
+import org.isf.medicalstock.model.Lot;
+import org.isf.medicalstock.service.LotIoOperationRepository;
+import org.isf.medicalstock.test.TestLot;
+import org.isf.medtype.model.MedicalType;
+import org.isf.medtype.service.MedicalTypeIoOperationRepository;
+import org.isf.medtype.test.TestMedicalType;
+import org.isf.utils.exception.OHException;
+import org.isf.ward.model.Ward;
+import org.isf.ward.service.WardIoOperationRepository;
+import org.isf.ward.test.TestWard;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
```

Comment:
Please use Junit 5 versions; we are moving to Junit 5 and this would help make one less class to convert.